python/invoke_release/tasks.py

```diff
@@ -1503,3 +1512,33 @@ def wheel(_):
         archive_name=archive_name,
         base_dir=base_dir
     ))
+
+
+def open_pull_request(base, head, title, token):
+    remote = subprocess.check_output(
+        ['git', 'remote', 'get-url', 'origin'],
+        stderr=subprocess.STDOUT,
+        )
+    repo = (remote.split(':')[1].split('.')[0])
+    url = 'https://api.github.com/repos/{repo}/pulls'.format(repo)
```

Comment:
You don't need to name the param in here, it can just be `'https://api.github.com/repos/{}/pulls'`